javascriptlib/src/mill/javascriptlib/TestModule.scala

```diff
@@ -5,7 +5,7 @@ import mill.*
 trait TestModule extends TaskModule {
   import TestModule.TestResult
 
-  def testForked(args: String*): Command[TestResult] =
+  def test(args: String*): Command[TestResult] =
```

Comment:
Why is this renamed?

Reply:
I think it was overwritten when I merged main into my branch, I have been working on an outdated branch for almost a month